### ISSUE TITLE
Windows policies. Strange (?) Behaviour

### ISSUE THREAD

**Issue by x1ongma0** [saltstack/salt#63319]
Hi, I am currently implementing CIS benchmark policies for Windows 2019 on salt stack.l (both by lgpo and registry edit).
I am facing some issues on the salt master while applying state files, namely some characters seem to be translated into the corresponding Hex code.
The weird fact is that this characters (an hyphen, for example) are part of a commented line (leading #).
The second "problem" is the following: when those weird behaviours are resolved (just deleting the offending chars one by one  in yaml comments), but new "generic" errors are raised (e.g. a bad char somewhere in a file of about 5k lines), is there a way to get some more specific feedback from salt master while applying states (-l debug doesn't help)?
Please, consider I am first checking all files through a yaml parser but...who knows?
I am a newbie with salt, so my questions may be "silly", but since I would really like to implement config management of servers with salt stack, any suggestion would be much appreciated.
Thank you in advance!

**Comment by twangboy** [comment]
Hey @x1ongma0 , thanks for reporting this. Would you mind posting the full stacktrace for the error you're seeing? Also a versions report for the version of salt on the minion (`salt-call --versions`)?

**Comment by x1ongma0** [comment]
Hi @twangboy, thank you for the reply. Below the requested information.
[salt_versions.txt](https://github.com/saltstack/salt/files/10235084/salt_versions.txt)
[cis_level1_common.sls.txt](https://github.com/saltstack/salt/files/10235091/cis_level1_common.sls.txt)
[trace.log.txt](https://github.com/saltstack/salt/files/10235092/trace.log.txt)

**Comment by x1ongma0** [comment]
Hi @twangboy . I keep on checking the state file. I am not sure to have captured all possibile syntax mistakes (my fault!), but the following are sure culprits, since I had to:
1. insert "-computer policy:" after "lgpo.set";
2. add escape char "\" before special chars as in "Advanced Audit Policy Configuration\\Audit Policies", etc.

**Comment by x1ongma0** [comment]
Last update @twangboy 
Eventually the upload file return only the following error (trace file updated):
    Rendering SLS 'base:win2019.cis_server_common.cis_level1_common' failed: while parsing a block mapping
  in "<unicode string>", line 6, column 11
did not find expected key
  in "<unicode string>", line 298, column 13

**Comment by twangboy** [comment]
I wonder if it has to do with different indent lengths... when I removed your comments from between the settings, I noticed that there is different indentation where you start to see the error:

```
Local Group Policy Server Common Level 1:
    lgpo.set:
        - computer_policy:
              "Enforce password history": "24"
              "Maximum password age": "60"
              "Minimum password age": "1"
              "Minimum password length": "14"
              "Password must meet complexity requirements": "Enabled"
              "Store passwords using reversible encryption": "Disabled"
            "Account lockout duration": "15"
            "Account lockout threshold": "10"
            "Reset account lockout counter after": "15"
            "Access Credential Manager as a trusted caller": ""
```
I think you probably need to have all those setting at the same indent level.

**Comment by x1ongma0** [comment]
Hi @twangboy. Yeah, the problem was definetly in the indent level and I solved everything with a decent text editor with saltstack/jinja2 syntax checker. Without such an editor it can be a daunting task to get a perfect yaml file (for example, I found other tiny - but blocking - errors, such an extra single space in a couple of lines).
I'll proceed now in fixing on my own the last few problems (I know how to handle them).
Again, thank you for your support!
